### PR TITLE
wX: Use template like strings

### PR DESCRIPTION
**Pull request by le-jeu**
Main idea is to be more generic: `wX(<NAME>, {placeholder1: value1,...})` allowing more elaborated strings.

Note: I found and removed some duplicates in the current locales.
Note2: some keys were renamed without updating other locales, current changes should now respect old translation

### CODE REVIEW

**Comment by cloudkucooland** [comment]
I like the concept. Thank you for this work.

File: src/code/translations/french.json [hunk 1]
```diff
@@ -81,216 +81,216 @@
...
-  "SET_3_PORT": "S\u00e9lectionner d'abord trois portails",
+  "SET_3_PORT": "Sélectionner d'abord trois portails",
   "SET_COMMENT": "Entrer un commentaire",
   "SET_LCOMMENT": "Commentaires du liens",
   "SET_LINK_COMMENT": "Entrer un commentaire pour le lien : ",
   "SET_MARKER_COMMENT": "Entrer un commentaire pour le marqueur : ",
-  "SET_MCOMMENT": "Commentaire pour ce marqueur : ${value}",
-  "SET_NEW_OP": "Donner un nom \u00e0 cette nouvelle op\u00e9ration",
-  "SET_PCOMMENT": "Entrer un commentaire pour ce portail : ${value}",
+  "SET_MCOMMENT": "Commentaire pour ce marqueur : {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:291]:
```suggestion
  "SET_MCOMMENT": "Commentaire pour ce marqueur : {portalName}",
```

File: src/code/translations/german.json [hunk 2]
```diff
@@ -147,9 +145,9 @@
     "SQUAD": "Kader",
     "LOC_UPDATE": "Standort aktualisieren",
     "SET_LCOMMENT": "Setze Kommentar für Link: ",
-    "SET_MCOMMENT": "Setze Kommentar für Markierung: ${value}",
-    "SET_PCOMMENT": "Setze Kommentar für Portal: ${value}",
-    "TEAM_CREATED": "Team ${value} erstellt.",
+    "SET_MCOMMENT": "Setze Kommentar für Markierung: {name}",
```
Comment (ruharen) [inline src/code/translations/german.json:148]:
```suggestion
    "SET_MCOMMENT": "Setze Kommentar für Markierung: {portalName}",
```

File: src/code/translations/italian.json [hunk 3]
```diff
@@ -288,21 +287,21 @@
...
+  "SEND TARGET CONFIRM": "Vuoi inviare il target {portalName} a {agent}?",
   "SET": "impostato",
   "SET_3_PORT": "Imposta prima i tre portali",
   "SET_COMMENT": "Imposta Commento",
   "SET_LINK_COMMENT": "Imposta commento per il link: ",
   "SET_LCOMMENT": "Imposta commento Link",
   "SET_MARKER_COMMENT": "Imposta commento per il marker su: ",
   "SET_MARKER_TYPE_TITLE": "Cambia tipo marker",
-  "SET_MCOMMENT": "Imposta commento Marker: ${value}",
+  "SET_MCOMMENT": "Imposta commento Marker: {name}",
```
Comment (ruharen) [inline src/code/translations/italian.json:298]:
```suggestion
  "SET_MCOMMENT": "Imposta commento Marker: {portalName}",
```

File: src/code/translations/spanish.json [hunk 4]
```diff
@@ -185,39 +185,38 @@
...
-  "SEND TARGET CONFIRM": "¿Quieres enviar el ${value} objetivo a ${option}?",
+  "SEND TARGET CONFIRM": "¿Quieres enviar el {portalName} objetivo a {agent}?",
   "SET": "definir",
   "SET_3_PORT": "¡Por favor, primero define los tres portales!",
   "SET_COMMENT": "Definir Comentario",
   "SET_LCOMMENT": "Definir Comentario de Link",
   "SET_LINK_COMMENT": "Definir comentario para el link",
   "SET_MARKER_COMMENT": "Definir comentario para el marcador en",
-  "SET_MCOMMENT": "Definir Comentario de Marcador ${value}",
+  "SET_MCOMMENT": "Definir Comentario de Marcador {name}",
```
Comment (ruharen) [inline src/code/translations/spanish.json:217]:
```suggestion
  "SET_MCOMMENT": "Definir Comentario de Marcador {portalName}",
```

File: src/code/translations/english.json [hunk 5]
```diff
@@ -299,21 +299,21 @@
...
   "SET_LINK_COMMENT": "Set comment for link: ",
   "SET_LCOMMENT": "Set Link Comment",
   "SET_MARKER_COMMENT": "Set comment for marker on: ",
   "SET_MARKER_TYPE_TITLE": "Change marker type",
-  "SET_MCOMMENT": "Set Marker Comment: ${value}",
+  "SET_MCOMMENT": "Set Marker Comment: {name}",
   "SET_NEW_OP": "Please Set the New Operation Name",
   "SET_PORT_COMMENT": "Set comment for portal: ",
-  "SET_PCOMMENT": "Set Portal Comment: ${value}",
+  "SET_PCOMMENT": "Set Portal Comment: {name}",
```
Comment (ruharen) [inline src/code/translations/english.json:313]:
```suggestion
  "SET_PCOMMENT": "Set Portal Comment: {portalName}",
```

File: src/code/translations/filipino.json [hunk 6]
```diff
@@ -235,17 +235,17 @@
...
     "SET_3_PORT": "Pumili muna ng tatlong portals!",
     "SET_COMMENT": "Magatalaga ng Komentayro",
     "SET_LCOMMENT": "Italaga ang commento sa Link",
     "SET_LINK_COMMENT": "Komento para sa link: ",
     "SET_MARKER_COMMENT": "Komento para sa marker: ",
-    "SET_MCOMMENT": "Italaga ang commento sa Marker: ${value}",
+    "SET_MCOMMENT": "Italaga ang commento sa Marker: {name}",
     "SET_NEW_OP": "Bigyan ng pangalan ang bagong operasyon",
-    "SET_PCOMMENT": "Italaga ang commento sa Portal: ${value}",
+    "SET_PCOMMENT": "Italaga ang commento sa Portal: {name}",
```
Comment (ruharen) [inline src/code/translations/filipino.json:248]:
```suggestion
    "SET_PCOMMENT": "Italaga ang commento sa Portal: {portalName}",
```

File: src/code/translations/french.json [hunk 7]
```diff
@@ -81,216 +81,216 @@
...
+  "SET_NEW_OP": "Donner un nom à cette nouvelle opération",
+  "SET_PCOMMENT": "Entrer un commentaire pour ce portail : {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:293]:
```suggestion
  "SET_PCOMMENT": "Entrer un commentaire pour ce portail : {portalName}",
```

File: src/code/translations/german.json [hunk 8]
```diff
@@ -147,9 +145,9 @@
...
+    "SET_PCOMMENT": "Setze Kommentar für Portal: {name}",
```
Comment (ruharen) [inline src/code/translations/german.json:149]:
```suggestion
    "SET_PCOMMENT": "Setze Kommentar für Portal: {portalName}",
```

File: src/code/translations/italian.json [hunk 9]
```diff
@@ -288,21 +287,21 @@
...
   "SET_NEW_OP": "Impostare il nuovo Nome Operazione",
   "SET_PORT_COMMENT": "Imposta il commento per il Portale: ",
-  "SET_PCOMMENT": "Imposta commento Portale: ${value}",
+  "SET_PCOMMENT": "Imposta commento Portale: {name}",
```
Comment (ruharen) [inline src/code/translations/italian.json:301]:
```suggestion
  "SET_PCOMMENT": "Imposta commento Portale: {portalName}",
```

File: src/code/translations/spanish.json [hunk 10]
```diff
@@ -185,39 +185,38 @@
...
   "SET_NEW_OP": "Por favor, define el Nuevo Nombre de la Operación",
-  "SET_PCOMMENT": "Definir Comentario de Portal ${value}",
+  "SET_PCOMMENT": "Definir Comentario de Portal {name}",
```
Comment (ruharen) [inline src/code/translations/spanish.json:219]:
```suggestion
  "SET_PCOMMENT": "Definir Comentario de Portal {portalName}",
```

File: src/code/translations/english.json [hunk 11]
```diff
@@ -343,9 +343,9 @@
   "NEWOP BUTTON": "New Op",
   "TARGET SENT": "Target sent",
   "TEAM": "Team",
-  "TEAM_CREATED": "Team ${value} created",
+  "TEAM_CREATED": "Team {name} created",
```
Comment (ruharen) [inline src/code/translations/english.json:346]:
```suggestion
  "TEAM_CREATED": "Team {teamName} created",
```

File: src/code/translations/filipino.json [hunk 12]
```diff
@@ -266,8 +266,8 @@
     "SYNC": "I-Download ang Matatamo na mga Operasyon",
     "TARDEST_DIFF": "Dapat magkaiba ang simula at destinasyon na portals.",
     "TARGET SENT": "Naipadala ang Target",
-    "TEAM PERM DENIED": "Walang pahintulot sa pangkat ${value}",
-    "TEAM_CREATED": "Pangkat ${value} nabuo",
+    "TEAM PERM DENIED": "Walang pahintulot sa pangkat {error}",
+    "TEAM_CREATED": "Pangkat {name} nabuo",
```
Comment (ruharen) [inline src/code/translations/filipino.json:270]:
```suggestion
    "TEAM_CREATED": "Pangkat {teamName} nabuo",
```

File: src/code/translations/french.json [hunk 13]
```diff
@@ -302,61 +302,61 @@
...
-  "TEAM": "\u00c9quipe",
-  "TEAMS BUTTON TITLE": "Liste des \u00e9quipes",
-  "TEAMS BUTTON": "\u00c9quipes",
-  "TEAM_CREATED": "\u00c9quipe ${value} cr\u00e9\u00e9e",
-  "TEAM_NAME": "Nom d'\u00e9quipe",
+  "TEAM PERM DENIED": "Permission refusée pour l'équipe : {error}",
+  "TEAM": "Équipe",
+  "TEAMS BUTTON TITLE": "Liste des équipes",
+  "TEAMS BUTTON": "Équipes",
+  "TEAM_CREATED": "Équipe {name} créée",
```
Comment (ruharen) [inline src/code/translations/french.json:316]:
```suggestion
  "TEAM_CREATED": "Équipe {teamName} créée",
```

File: src/code/translations/german.json [hunk 14]
```diff
@@ -147,9 +145,9 @@
...
+    "TEAM_CREATED": "Team {name} erstellt.",
```
Comment (ruharen) [inline src/code/translations/german.json:150]:
```suggestion
    "TEAM_CREATED": "Team {teamName} erstellt.",
```

File: src/code/translations/english.json [hunk 15]
```diff
@@ -87,23 +87,23 @@
   "CLEAROPS PROMPT": "This will clear all OPS and data related to Wasabee. Everything will be restored from the server at next sync.",
   "CLOSE": "Close",
   "COMPLETED": "Completed",
-  "COMPLETED BY": "Completed by ${value}",
-  "CON_DEL": "Confirm Delete: ${value}",
+  "COMPLETED BY": "Completed by {name}",
```
Comment (ruharen) [inline src/code/translations/english.json:90]:
```suggestion
  "COMPLETED BY": "Completed by {agentName}",
```

File: src/code/translations/filipino.json [hunk 16]
```diff
@@ -56,16 +56,16 @@
     "CLEAROPS BUTTON": "Burahin Local Ops",
     "CLEAROPS PROMPT": "Nais mo bang tangalin ang lahat ng opersyon sa local storage? Ang mga Ops na nakalagay sa server ay mababawi sa susunod na sync.",
     "COMMENT": "Komentaryo",
-    "COMPLETED BY": "Nakumpleto ni ${value}",
+    "COMPLETED BY": "Nakumpleto ni {name}",
```
Comment (ruharen) [inline src/code/translations/filipino.json:59]:
```suggestion
    "COMPLETED BY": "Nakumpleto ni {agentName}",
```

File: src/code/translations/french.json [hunk 17]
```diff
@@ -26,53 +26,53 @@
...
-  "CLEAROPS PROMPT": "Toutes les op\u00e9rations et donn\u00e9es relatives \u00e0 Wasabee seront supprim\u00e9es de cet appareil. Seules les donn\u00e9es pr\u00e9sentes sur le/les serveurs seront r\u00e9cup\u00e9r\u00e9es \u00e0 la prochaine synchronisation.",
+  "CLEAROPS BUTTON TITLE": "Supprimer TOUTES les données Wasabee",
+  "CLEAROPS BUTTON": "Supprimer les données Wasabee",
+  "CLEAROPS PROMPT": "Toutes les opérations et données relatives à Wasabee seront supprimées de cet appareil. Seules les données présentes sur le/les serveurs seront récupérées à la prochaine synchronisation.",
   "CLEAR_EVERYTHING": "Supprimer Portails/Links et Marquers",
   "CLOSE": "Fermer",
   "COMMENT": "Commentaire",
-  "COMPLETED BY": "Fait par ${value}",
-  "COMPLETED": "Complet\u00e9e",
+  "COMPLETED BY": "Fait par {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:65]:
```suggestion
  "COMPLETED BY": "Fait par {agentName}",
```

File: src/code/translations/german.json [hunk 18]
```diff
@@ -24,7 +24,7 @@
     "LA": "L8+ einige LA",
     "LA DESC": "Ausgehend von der Anzahl und Typ von Link-Amps könnte ein niedrigeres Portallevel ausreichen",
     "ASSIGNED TO": "Zugewiesen an",
-    "COMPLETED BY": "Abgeschlossen von ${value}",
+    "COMPLETED BY": "Abgeschlossen von {name}",
```
Comment (ruharen) [inline src/code/translations/german.json:27]:
```suggestion
    "COMPLETED BY": "Abgeschlossen von {agentName}",
```

File: src/code/translations/italian.json [hunk 19]
```diff
@@ -87,23 +86,23 @@
   "CLEAROPS PROMPT": "Questo eliminerà tutte le OP e i dati relativi a Wasabee. Saranno ripristinati alla prossima sincronizzazione.",
   "CLOSE": "Chiudi",
   "COMPLETED": "Completato",
-  "COMPLETED BY": "Completato da ${value}",
-  "CON_DEL": "Conferma eliminazione: ${value}",
+  "COMPLETED BY": "Completato da {name}",
```
Comment (ruharen) [inline src/code/translations/italian.json:89]:
```suggestion
  "COMPLETED BY": "Completato da {agentName}",
```

File: src/code/translations/spanish.json [hunk 20]
```diff
@@ -48,21 +48,21 @@
   "CLEAROPS BUTTON TITLE": "Borrar TODOS los Datos de Wasabee",
   "CLEAROPS PROMPT": "Esta opción borrará todas las OPS y datos relacionados a Wasabee. Todo será restaurado desde el servidor en la próxima sincronización.",
   "COMPLETED": "Completo",
-  "COMPLETED BY": "Completado por ${value}",
-  "CON_DEL": "Confirmar Borrado ${value}",
+  "COMPLETED BY": "Completado por {name}",
```
Comment (ruharen) [inline src/code/translations/spanish.json:51]:
```suggestion
  "COMPLETED BY": "Completado por {agentName}",
```

File: src/code/translations/english.json [hunk 21]
```diff
@@ -116,7 +116,7 @@
   "EXPORT": "Export: ",
   "EXPORT OP": "Export Op",
   "EXPORT OP TITLE": "Export current Op",
-  "FAKED": "Faked: [${value}]",
+  "FAKED": "Faked: [{id}]",
```
Comment (ruharen) [inline src/code/translations/english.json:119]:
```suggestion
  "FAKED": "Faked: [{portalId}]",
```

File: src/code/translations/french.json [hunk 22]
```diff
@@ -81,216 +81,216 @@
...
+  "D_SHOW_LIST": "Entrer les clées défensive",
   "DestroyPortalAlert": "Casse",
   "END_PORT": "Portail de fin",
-  "EXPORT OP TITLE": "Exporter l'op\u00e9ration",
+  "EXPORT OP TITLE": "Exporter l'opération",
   "EXPORT OP": "Exporter",
   "EXPORT": "Exporter : ",
   "ExcludeMarker": "Exclure d'Auto-Draw/Mark",
-  "FAKED": "Faux [${value}]",
+  "FAKED": "Faux [{id}]",
```
Comment (ruharen) [inline src/code/translations/french.json:99]:
```suggestion
  "FAKED": "Faux [{portalId}]",
```

File: src/code/translations/italian.json [hunk 23]
```diff
@@ -116,7 +115,7 @@
   "EXPORT": "Esporta: ",
   "EXPORT OP": "Esporta Op",
   "EXPORT OP TITLE": "Esporta Op attuale",
-  "FAKED": "Simulato: [${value}]",
+  "FAKED": "Simulato: [{id}]",
```
Comment (ruharen) [inline src/code/translations/italian.json:118]:
```suggestion
  "FAKED": "Simulato: [{portalId}]",
```

File: src/code/translations/english.json [hunk 24]
```diff
@@ -162,7 +162,7 @@
   "LINK ASSIGNMENT": " Link Assignment",
   "LINKS BUTTON TITLE": "Links",
   "LOADING": "[loading]",
-  "LOADING1": "Loading: [${value}]",
+  "LOADING1": "Loading: [{guid}]",
```
Comment (ruharen) [inline src/code/translations/english.json:165]:
```suggestion
  "LOADING1": "Loading: [{portalGuid}]",
```

File: src/code/translations/french.json [hunk 25]
```diff
@@ -81,216 +81,216 @@
...
+  "LINK ASSIGNMENT": "Assigner à",
   "LINK STATE PROMPT": "Statut du lien",
   "LINK STATE": "Statut",
   "LINKS BUTTON TITLE": "Liens",
   "LINKS": "Liens",
   "LINKS2": ": Liens",
   "LOAD PORTALS": "Charger les portails",
   "LOADING": "[chargement]",
-  "LOADING1": "Chargement: ${value}",
+  "LOADING1": "Chargement: {guid}",
```
Comment (ruharen) [inline src/code/translations/french.json:149]:
```suggestion
  "LOADING1": "Chargement: {portalGuid}",
```

File: src/code/translations/italian.json [hunk 26]
```diff
@@ -162,7 +161,7 @@
   "LINK ASSIGNMENT": " Assegnamento Link",
   "LINKS BUTTON TITLE": "Link",
   "LOADING": "[caricamento]",
-  "LOADING1": "Caricamento: [${value}]",
+  "LOADING1": "Caricamento: [{guid}]",
```
Comment (ruharen) [inline src/code/translations/italian.json:164]:
```suggestion
  "LOADING1": "Caricamento: [{portalGuid}]",
```

File: src/code/translations/english.json [hunk 27]
```diff
@@ -87,23 +87,23 @@
...
+  "COMPLETED BY": "Completed by {name}",
+  "CON_DEL": "Confirm Delete: {name}",
   "COMMENT": "Comment",
   "CONFIRM_DELETE": "Do you really want to delete this link: ",
   "COUNT": "Count",
   "CREATE_NEW_TEAM": "Create New Team",
   "CUR_USER_INFO": "Current User Information",
   "DT_FORMAT": "Draw Tools Format",
-  "DEFAULT OP NAME": "New Op: ${value}",
+  "DEFAULT OP NAME": "New Op: {name}",
```
Comment (ruharen) [inline src/code/translations/english.json:98]:
```suggestion
  "DEFAULT OP NAME": "New Op: {opName}",
```

File: src/code/translations/filipino.json [hunk 28]
```diff
@@ -56,16 +56,16 @@
...
-    "CON_DEL": "Kumpirmahin ang Pagtangal: ${value}",
+    "CON_DEL": "Kumpirmahin ang Pagtangal: {name}",
     "CONFIRM_DELETE": "Gusto mo bang tangalin ang link: ",
     "COUNT": "Bilang",
     "CREATE_NEW_TEAM": "Magbuo ng bagong Pangkat",
     "CreateLinkAlert": "Link",
     "CUR_USER_INFO": "Kasalukuyang Impormasyon ng User",
     "D_SHOW_LIST": "Ipasok ang Defensive Keys",
-    "DEFAULT OP NAME": "Bagong Op ${value}",
+    "DEFAULT OP NAME": "Bagong Op {name}",
```
Comment (ruharen) [inline src/code/translations/filipino.json:68]:
```suggestion
    "DEFAULT OP NAME": "Bagong Op {opName}",
```

File: src/code/translations/french.json [hunk 29]
```diff
@@ -26,53 +26,53 @@
...
+  "CON_DEL": "Confirmer la suppression : {name}",
   "COUNT": "Nombre",
-  "CREATE_NEW_TEAM": "Cr\u00e9er une \u00e9quipe",
+  "CREATE_NEW_TEAM": "Créer une équipe",
   "CUR_USER_INFO": "Information sur l'utilisateur courant",
   "CapturePortalMarker": "Capture",
   "CreateLinkAlert": "Lien",
-  "DEFAULT OP NAME": "Op\u00e9ration ${value}",
-  "DELETE ANCHOR PROMPT": "Voulez vous supprimer cette ancre et tous les liens associ\u00e9s : ",
+  "DEFAULT OP NAME": "Opération {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:74]:
```suggestion
  "DEFAULT OP NAME": "Opération {opName}",
```

File: src/code/translations/german.json [hunk 30]
```diff
@@ -37,19 +37,18 @@
     "QD TITLE": "Schnelle Layer Zeichnung",
     "QD BUTTON END": "Klicken um das Feld zeichnen zu beenden",
     "QD END": "Ende",
-    "DEFAULT OP NAME": "Neue Operation: ${value}",
+    "DEFAULT OP NAME": "Neue Operation: {name}",
```
Comment (ruharen) [inline src/code/translations/german.json:40]:
```suggestion
    "DEFAULT OP NAME": "Neue Operation: {opName}",
```

File: src/code/translations/spanish.json [hunk 31]
```diff
@@ -48,21 +48,21 @@
...
+  "CON_DEL": "Confirmar Borrado {name}",
   "COMMENT": "Comentario",
   "CONFIRM_DELETE": "Estás seguro de querer borrar este link",
   "COUNT": "Contar",
   "CREATE_NEW_TEAM": "Crear Nuevo Equipo",
   "CUR_USER_INFO": "Información Actual de Usuario",
   "DT_FORMAT": "Formato Draw Tools",
-  "DEFAULT OP NAME": "Nueva Op: ${value}",
+  "DEFAULT OP NAME": "Nueva Op: {name}",
```
Comment (ruharen) [inline src/code/translations/spanish.json:59]:
```suggestion
  "DEFAULT OP NAME": "Nueva Op: {opName}",
```

File: src/code/translations/italian.json [hunk 32]
```diff
@@ -87,23 +86,23 @@
...
+  "COMPLETED BY": "Completato da {name}",
+  "CON_DEL": "Conferma eliminazione: {name}",
   "COMMENT": "Commento",
   "CONFIRM_DELETE": "Vuoi davvero eliminare questo link: ",
   "COUNT": "Conteggio",
   "CREATE_NEW_TEAM": "Crea Nuovo Team",
   "CUR_USER_INFO": "Informazioni utente attuale",
   "DT_FORMAT": "Formato Draw Tools",
-  "DEFAULT OP NAME": "Nuova Op: ${value}",
+  "DEFAULT OP NAME": "Nuova Op: {name}",
```
Comment (ruharen) [inline src/code/translations/italian.json:97]:
```suggestion
  "DEFAULT OP NAME": "Nuova Op: {opName}",
```

File: src/code/translations/french.json [hunk 33]
```diff
@@ -81,216 +81,216 @@
...
-  "KEY_LIST2": "Liste des cl\u00e9es pour l'op\u00e9ration : ${value}",
-  "KNOWN_BLOCK": "Liens bloquants connus : ${value}",
-  "LA DESC": "Le niveau n\u00e9cessaire du portail peut \u00eatre plus faible en fonction du type des Link Amp utilis\u00e9s.",
+  "INPUT_DT_KEY_COUNT": "Entrer le nombre de clés défensive",
+  "INPUT_SQUAD_NAME": "Entrer le nom de l'équipe",
+  "INVALID REQUEST": "Requête invalide",
+  "KEYS": "Clées",
+  "KEY_LIST": "Listes des clées pour le portail : {opName}",
+  "KEY_LIST2": "Liste des clées pour l'opération : {name}",
+  "KNOWN_BLOCK": "Liens bloquants connus : {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:136]:
```suggestion
  "KNOWN_BLOCK": "Liens bloquants connus : {opName}",
```

File: src/code/translations/french.json [hunk 34]
```diff
@@ -81,216 +81,216 @@
...
+  "OP PERM DENIED": "Permission refusée pour l'opération : {opID}",
+  "OPERATIONS": "Opérations",
+  "OPER_COLOR": "Couleur de l'opération",
+  "OPER_NAME": "Nom de l'opération",
+  "OPS BUTTON TITLE": "Opérations",
   "OPS BUTTON": "Ops",
   "OPTION": "Options",
-  "OP_CHECKLIST": "Operation Checklist: ${value}",
-  "OP_NAME_UNSET": "Le nom de l'op\u00e9ration n'a pas \u00e9t\u00e9 d\u00e9fini",
+  "OP_CHECKLIST": "Operation Checklist: {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:225]:
```suggestion
  "OP_CHECKLIST": "Operation Checklist: {opName}",
```

File: src/code/translations/french.json [hunk 35]
```diff
@@ -81,216 +81,216 @@
...
-  "HOURS": "(il y a ${value} heures)",
+  "HG": "Field homogène",
+  "HOURS": "(il y a {hours} heures)",
   "HOW_TO_VIDS": "<h3>Tuto:</h3><div class='desc'><a href='https://www.youtube.com/playlist?list=PLyku9nmtwrADKQM9_EZk7NYbZXVOrBhXa'>Wasabee Playlist</a></div>",
-  "IMPORT_OP": "Importer l'op\u00e9ration",
-  "IMPORT_OP_SUCCESS": "Importer l'op\u00e9ration : ${value} Reussi.",
-  "IMPORT_OP_TITLE": "Importer l'op\u00e9ration : ${value}",
+  "IMPORT_OP": "Importer l'opération",
+  "IMPORT_OP_SUCCESS": "Importer l'opération : {name} Reussi.",
+  "IMPORT_OP_TITLE": "Importer l'opération : {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:123]:
```suggestion
  "IMPORT_OP_TITLE": "Importer l'opération : {opName}",
```

File: src/code/dialogs/importDialog.js [hunk 36]
```diff
@@ -142,7 +142,7 @@ const ImportDialog = WDialog.extend({
       const importedOp = new WasabeeOp(data);
       await importedOp.store();
       await makeSelectedOperation(importedOp.ID);
-      alert(wX("IMPORT_OP_SUCCESS", importedOp.name));
+      alert(wX("IMPORT_OP_SUCCESS", { name: importedOp.name }));
```
Comment (ruharen) [inline src/code/dialogs/importDialog.js:145]:
```suggestion
      alert(wX("IMPORT_OP_SUCCESS", { opName: importedOp.name }));
```

File: src/code/translations/english.json [hunk 37]
```diff
@@ -134,13 +134,13 @@
...
-  "HOURS": " (${value} hours ago)",
+  "HOURS": " ({hours} hours ago)",
   "HOW_TO_VIDS": "<h3>How-To Videos:</h3><div class='desc'><a href='https://www.youtube.com/playlist?list=PLyku9nmtwrADKQM9_EZk7NYbZXVOrBhXa'>Wasabee Playlist</a></div>",
   "IMP_COMP": "Import Complete. Found ",
   "IMP_DT_OP": "Imported Drawtools Op: ",
   "IMPORT_OP": "Import Operation",
-  "IMPORT_OP_TITLE": "Import Op: ${value}",
-  "IMPORT_OP_SUCCESS": "Imported Operation: ${value} successfuly.",
+  "IMPORT_OP_TITLE": "Import Op: {name}",
+  "IMPORT_OP_SUCCESS": "Imported Operation: {name} successfuly.",
```
Comment (ruharen) [inline src/code/translations/english.json:143]:
```suggestion
  "IMPORT_OP_SUCCESS": "Imported Operation: {opName} successfuly.",
```

File: src/code/translations/french.json [hunk 38]
```diff
@@ -81,216 +81,216 @@
...
-  "HG": "Field homog\u00e8ne",
-  "HOURS": "(il y a ${value} heures)",
+  "HG": "Field homogène",
+  "HOURS": "(il y a {hours} heures)",
   "HOW_TO_VIDS": "<h3>Tuto:</h3><div class='desc'><a href='https://www.youtube.com/playlist?list=PLyku9nmtwrADKQM9_EZk7NYbZXVOrBhXa'>Wasabee Playlist</a></div>",
-  "IMPORT_OP": "Importer l'op\u00e9ration",
-  "IMPORT_OP_SUCCESS": "Importer l'op\u00e9ration : ${value} Reussi.",
-  "IMPORT_OP_TITLE": "Importer l'op\u00e9ration : ${value}",
+  "IMPORT_OP": "Importer l'opération",
+  "IMPORT_OP_SUCCESS": "Importer l'opération : {name} Reussi.",
```
Comment (ruharen) [inline src/code/translations/french.json:122]:
```suggestion
  "IMPORT_OP_SUCCESS": "Importer l'opération : {opName} Reussi.",
```

File: src/code/translations/german.json [hunk 39]
```diff
@@ -159,8 +157,8 @@
     "NO_DT_ITEMS": "Keine DrawTools Daten erkannt",
     "ONLY_DT_IMP": " (nur für DrawTools imports)",
     "NO_STOCK_INTEL": "Wasabee unterstützt keine Standard Intel imports",
-    "IMPORT_OP_TITLE": "Importierte Op: ${value}",
-    "IMPORT_OP_SUCCESS": "Importierte Operation: ${value}. Erfolgreich.",
+    "IMPORT_OP_TITLE": "Importierte Op: {name}",
+    "IMPORT_OP_SUCCESS": "Importierte Operation: {name}. Erfolgreich.",
```
Comment (ruharen) [inline src/code/translations/german.json:161]:
```suggestion
    "IMPORT_OP_SUCCESS": "Importierte Operation: {opName}. Erfolgreich.",
```

File: src/code/translations/french.json [hunk 40]
```diff
@@ -81,216 +81,216 @@
...
   "PENDING": "En attente",
-  "PERMS": "Permissions de ${value}",
-  "PICK_DEST_FIRST": "Vous devez d'abord s\u00e9lectionner un portail de destination!",
-  "PICK_TAR_FIRST": "Vous devez d'abord s\u00e9lectionner un portail cible!",
+  "PERMS": "Permissions de {opName}",
+  "PICK_DEST_FIRST": "Vous devez d'abord sélectionner un portail de destination!",
+  "PICK_TAR_FIRST": "Vous devez d'abord sélectionner un portail cible!",
   "PLEASE_SELECT_PORTAL": "Selectionner un portail",
-  "PORTAL KEY LIST": "Cl\u00e9s du portail ${value}",
+  "PORTAL KEY LIST": "Clés du portail {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:235]:
```suggestion
  "PORTAL KEY LIST": "Clés du portail {portalName}",
```

File: src/code/translations/french.json [hunk 41]
```diff
@@ -81,216 +81,216 @@
...
-  "KEY_LIST": "Listes des cl\u00e9es pour le portail : ${value}",
-  "KEY_LIST2": "Liste des cl\u00e9es pour l'op\u00e9ration : ${value}",
-  "KNOWN_BLOCK": "Liens bloquants connus : ${value}",
-  "LA DESC": "Le niveau n\u00e9cessaire du portail peut \u00eatre plus faible en fonction du type des Link Amp utilis\u00e9s.",
+  "INPUT_DT_KEY_COUNT": "Entrer le nombre de clés défensive",
+  "INPUT_SQUAD_NAME": "Entrer le nom de l'équipe",
+  "INVALID REQUEST": "Requête invalide",
+  "KEYS": "Clées",
+  "KEY_LIST": "Listes des clées pour le portail : {opName}",
+  "KEY_LIST2": "Liste des clées pour l'opération : {name}",
```
Comment (ruharen) [inline src/code/translations/french.json:135]:
```suggestion
  "KEY_LIST2": "Liste des clées pour l'opération : {opName}",
```

**Review by ruharen — CHANGES_REQUESTED** [review]
I've tried to give consistency for the word 'name' in translation parameters, changing it to opName, agentName, etc, as needed. It also improves disambiguation for translators: "{name}??, name of what??" ;)

File: src/code/dialogs/blockersList.js [hunk 42]
```diff
@@ -71,7 +71,7 @@ const BlockerList = WDialog.extend({
     };
 
     this.createDialog({
-      title: wX("KNOWN_BLOCK", operation.name),
+      title: wX("KNOWN_BLOCK", { name: operation.name }),
```
[the remark this answers is not in the document]
Comment (cloudkucooland) [reply src/code/dialogs/blockersList.js:74]: These all look good to me. I'll let LeJeu commit them as he reviews them.

**Pull request merged**